charts/nginx-website/values.yaml

```diff
@@ -61,3 +61,8 @@ nginx:
     autoindex: "on"
     ## Uncomment to enable try_files directive
     # tryFiles: $uri /index.html
+  redirections: {}
+    ## Example:
+    # - location: /plugin-installation-trend
+    #   returnedCode: 301
+    #   destination: https://raw.githubusercontent.com/jenkins-infra/infra-statistics/gh-pages/
```

Comment:
This isn't the same API and may still cause breaking changes.

1. There's no guarantee that consumers will follow redirects
2. raw.githubusercontent returns a content-type of text/plain which may break consumers